app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingSlideViewModel" />
```

Comment:
This view model is incorrect.

Reply:
In portrait all these details are set from strings.xml file directly but in this you are trying to use viewmodel (which is not necessary). The viewmodel should be used consistently in landscape and portrait layout.

Reply:
@rt4914 I had unassigned you as i was supposed to make changes once i complete will assign you back.

Reply:
Done